examples/decoding/plot_simulated_data.py

```diff
@@ -41,9 +41,9 @@ def create_simulation_data(snr=0, n_samples=2 * 100, size=12, random_state=1):
     w[-roi_size:, -roi_size:, 0:roi_size] = 0.5
```

Comment:
This is the only remaining part of the py2 py3 compatibility, as all print functions disapeared on rebase. This is needed because sphinx-gallery imports from future division, so it really requires integers as keys it crashes otherwise.